src/Phpro/SoapClient/BridgeBundle/Type/SoapCall.php

```diff
@@ -111,11 +115,16 @@ public function getTiming()
      */
     public function getRequestBody()
     {
-        $request = new \DOMDocument();
-        $request->preserveWhiteSpace = false;
-        $request->formatOutput = true;
-        $request->loadXML($this->requestBody);
-        return $request->saveXML();
+        if ($this->requestBody) {
```

Comment:
Can you invert the if and add an early return instead?
Is this actually an error, or should the message say something like 'Empty resuest'?